src/mapeo-project.js

```diff
@@ -840,6 +849,35 @@ async function deleteAll(dataType) {
   return Promise.all(deletions)
 }
 
+/**
+@param {Object} dataTypes
+@param {MapeoProject['translation']} dataTypes.translation
+@param {MapeoProject['preset']} dataTypes.presets
+@param {MapeoProject['field']} dataTypes.fields
```

Comment:
Nitpick for consistency:

```suggestion
 * @param {Object} dataTypes
 * @param {MapeoProject['translation']} dataTypes.translation
 * @param {MapeoProject['preset']} dataTypes.presets
 * @param {MapeoProject['field']} dataTypes.fields
```